sql/src/parser.rs

```diff
@@ -352,8 +352,39 @@ impl<'a> Parser<'a> {
                 );
             }
         }
+        let mut cols: Vec<ColumnDef> = vec![];
```

Comment:
I guess there is no need to reconstruct the `ColumnDef`s because there is no influence to keep the extra TimestampKey option in the `column.options`.

So I guess what we need here is just to build `TableConstraint` if possible.